go/internal/cryptoutil/cryptoutil.go

```diff
@@ -144,3 +165,85 @@ func EdwardsToMontgomeryPriv(privKey crypto.PrivKey) (*[KeySize]byte, error) {
 
 	return &mongPriv, nil
 }
+
+// AESGCMEncrypt use AES+GCM to encrypt plaintext data.
+//
+// The generated output will be longer than the original plaintext input.
+func AESGCMEncrypt(key, data []byte) ([]byte, error) {
```

Comment:
this helper is finally not used, but since it's generic and tested, I let it in the current helper package